Update template generation for tkn pac create repo command

savitaashture opened the pull request:
<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->

# Changes
Fixes: https://github.com/openshift-pipelines/pipelines-as-code/issues/930

**Result:**

##### Before change:
```
$ tkn-pac create repo

? Enter the Git repository url containing the pipelines (default: https://github.com/savitaashture/news-demo):  
? Please enter the namespace where the pipeline should run (default: news-demo-pipelines): 
! Namespace news-demo-pipelines is not found
? Would you like me to create the namespace news-demo-pipelines? Yes
✓ Repository savitaashture-news-demo has been created in news-demo-pipelines namespace
? There is already a file named: .tekton/pipelinerun.yaml would you like me to override it? Yes
✓ We have detected your repository using the programming language Go.
✓ A basic template has been created in /home/sashture/Go/src/github.com/savitaashture/news-demo/.tekton/pipelinerun.yaml, feel free to customize it.
ℹ You can test your pipeline manually with: tkn-pac resolve -f .tekton/pipelinerun.yaml | kubectl create -f-
🚀 You can use the command "tkn pac setup" to setup a repository with webhook

```
If there is file already exist a pop up message use to display whether to override existing file or not

##### After change:
```
$ ./tkn-pac create repo

? Enter the Git repository url containing the pipelines (default: https://github.com/savitaashture/news-demo):  
? Please enter the namespace where the pipeline should run (default: news-demo-pipelines): 
! Namespace news-demo-pipelines is not found
? Would you like me to create the namespace news-demo-pipelines? Yes
✓ Repository savitaashture-news-demo has been created in news-demo-pipelines namespace
ℹ There is already a file named: .tekton/pipelinerun.yaml, skipping template generation, feel free to use "tkn pac generate" command to generate sample template.
🚀 You can use the command "tkn pac setup" to setup a repository with webhook

```
If there is file already exist just display the info no need to pop message

<!-- Describe your changes here- ideally you can get that description straight from
your descriptive commit message(s)! -->

# Submitter Checklist

- [ ] ♽  Run `make test lint` before submitting a PR (ie: with [pre-commit](https://pipelinesascode.com/dev/tools), no need to waste CPU cycle on CI
- [ ] 📖 If you are adding a user facing feature or make a change of the behavior, please verify that you have documented it
- [ ] 🧪 100% coverage is not a target but most of the time we would rather have a unit test if you make a code change.
- [ ] 🎁 If that's something that is possible to do please ensure to check if we can add a e2e test.
- [ ] 🔎 If there is a flakiness in the CI tests then don't *necessary* ignore it, better get the flakyness fixed before merging or if that's not possible there is a good reason to bypass it. (token rate limitation may be a good reason to skip).

savitaashture committed:
Update tkn pac generate command

sm43 commented:
if a user is explicitly using `tkn pac generate` shouldn't we create it regardless what is in `.tekton`?
user might want to add a new  pipelinerun
`There is already a file named: f, skipping template generation, feel free to use "tkn pac generate" command to generate sample template.`
we are saying file exist but feel free to use the same command 😄 
not generating file in repo create command is okay but here I feel we should just create it.

savitaashture commented:
> if a user is explicitly using `tkn pac generate` shouldn't we create it regardless what is in `.tekton`? user might want to add a new pipelinerun `There is already a file named: f, skipping template generation, feel free to use "tkn pac generate" command to generate sample template.` we are saying file exist but feel free to use the same command smile not generating file in repo create command is okay but here I feel we should just create it.

Yes agree

Kept the behavior of `tkn pac generate` command as it is

Changed behavior only for `tkn pac create repo`

Updated description as well

sm43 commented:
/lgtm

chmouel commented:
/retest

chmouel commented:
it's weird that golangci-lint time out on the gcp cluster for this

chmouel commented:
oh that one is new 🤔 

![image](https://user-images.githubusercontent.com/98980/200302821-0d1f44d4-c593-4325-8ba4-c895a41ffbd9.png)

chmouel commented:
/retest

savitaashture commented:
/retest

savitaashture commented:
/retest
